docs/pages/reference/mesheryctl/mesheryctl-system-dashboard.md

```diff
@@ -70,6 +70,19 @@ mesheryctl system dashboard --skip-browser
 </div>
 </pre>
 
+## [Optional] Port-forward Meshery as Kubernetes Service
+
+Ensure [kubectl](https://kubernetes.io/docs/tasks/tools/) is installed on your local machine.
```

Comment:
@alphaX86 this change will be overwritten on next release of mesheryctl, right?

Reply:
I'm afraid it will 💀. Technically it would vanish once this PR is merged and CI script is triggered which is bad. We need to halt this PR for a while or make changes in other file if we need this note.

Reply:
If we do need this note, then we need to stop the CI Docs action for a while

Reply:
CLI-Docs auto generation will just over write this particular file right other file change wont have the impact right like aks.md, helm.md, eks.md.

Reply:
@leecalcote is there any other place where we can mention this info of port-forwarding

Reply:
@aboobakersiddiqr63, yes, you can essentially revert to your prior train of thought and include these instructions in this page - https://docs.meshery.io/tasks/accessing-meshery-ui

Reply:
@leecalcote will make the change and commit again :)